.github/workflows/deploy.yml

```diff
@@ -26,15 +26,20 @@ jobs:
       - uses: actions/checkout@v2.3.4
       - uses: actions/setup-node@v2.3.2
         with:
-          node-version: "12"
+          node-version: "14"
 
-      - name: Pull Docs
+      - name: Install Dependencies
         run: |
-          node sync-docs.js && git status
+          cd scripts && yarn install
+          cd ../website && yarn install
```

Comment:
```suggestion
          cd scripts && yarn
          cd ../website && yarn
```

Reply:
It seems that the effect of `yarn install` is the same as that of `yarn`, and `yarn install` is more explicit. 🤔

Reply:
Yes, they are equivalent.